### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by wm4**
Random unrelated minor changes (probably better than making 5 separate PRs).

### CODE REVIEW

**Comment by wm4** [comment]
Added 3 other not-so-minor changes, which affect --vf-defaults and some audio things.

File: DOCS/man/options.rst [hunk 1]
```diff
@@ -3453,6 +3443,46 @@ Software Scaler
...
+    entries. (default: no)
+
+``--audio-resample-normalize=<yes|no>``
+    Whether to normalize when remixing channel layouts (default: no).
+    ``auto`` uses the value set by ``--audio-normalize-downmix``.
+
+``--audio-normalize-downmix=<yes|no>``
+    Enable/disable normalization if surround audio is downmixed to stereo
+    (default: no). If this is disabled, downmix can cause clipping. If it's
+    enabled, the output might be too silent. It depends on the source audio.
```
Comment (kevmitch) [inline DOCS/man/options.rst]: "too quiet" sound better

File: DOCS/man/options.rst [hunk 2]
```diff
@@ -3453,6 +3443,46 @@ Software Scaler
...
+
+    Technically, this changes the ``normalize`` suboption of the
+    ``lavrresample`` audio filter, which performs the downmixing.
+
+    If downmix happens outside of mpv for some reason, this has no effect.
```
Comment (kevmitch) [inline DOCS/man/options.rst]: Might want to explicitly mention codec downmixing or system mixer.

**Comment by wm4** [comment]
Fixed the pointed out issues.

File: audio/filter/af_lavrresample.c [hunk 3]
```diff
@@ -149,6 +150,11 @@ static int af_open(struct af_instance *af)
 
     s->converter = mp_aconverter_create(af->global, af->log, &s->opts);
 
+    if (s->deprecation_warning) {
```
Comment (kevmitch) [inline audio/filter/af_lavrresample.c]: sorry, I didn't catch this earlier, but I incorrectly see this warning with 

    mpv  /home/kevmitch/mm/tmp/ZT.mkv  --config-dir=config --speed=1.1 --audio-pitch-correction=no

where `config/input.conf` contains 

    a cycle_values audio-device null auto

hit `a`

**Comment by wm4** [comment]
Should be fixed now.

**Pull request merged**